packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/specialheaders/models/RepeatabilityResult.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.cadl.specialheaders.models;
+
+/**
+ * Repeatability Result header options.
+ */
+public enum RepeatabilityResult {
+    /**
+     * If the request was accepted and the server guarantees that the server state reflects a single execution of the
+     * operation.
+     */
+    ACCEPTED("accepted"),
+
+    /**
+     * If the request was rejected because the combination of Repeatability-First-Sent and Repeatability-Request-ID were
+     * invalid
+     * or because the Repeatability-First-Sent value was outside the range of values held by the server.
+     */
+    REJECTED("rejected");
+
+    /**
+     * The actual serialized value for a RepeatabilityResult instance.
+     */
```

Comment:
Caused by [case1](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#1-responses-header-schema-has-public-access-in-tcgc)